modules/03 Front End Development/Assignments/54/main.js

```diff
@@ -0,0 +1,151 @@
+reStart = 0;
+function newGame() {
+    if (reStart === 1) {
+        document.body.removeChild(btn);
+    }
+    light = document.getElementById("light").checked;
+    dark = document.getElementById("dark").checked;
+    if (light === true) {
+        document.body.style.background = "linear-gradient(-90deg, #D8E2DC, #FFE5D9, #FFCAD4, #F4ACB7)";
+        event.target.style.background = "#D8E2DC";
+    }
+    if (dark === true) {
+        document.body.style.background = "linear-gradient(-90deg, #2B2D42, #8D99AE, #EF233C, #D90429)";
+        event.target.style.background = "#2B2D42";
+        event.target.style.color = "white";
+    }
+    document.getElementById("game-status").innerHTML = "";
+    document.getElementById("btn-start-game").disabled = true;
+    player1 = document.getElementById("player1").value + " turn";
+    player2 = document.getElementById("player2").value + " turn";
+    playerTurn = "X";
+    count = 0;
+    var btnCircle = document.getElementsByClassName("circle");
+    for (var i = 0; i < btnCircle.length; i++) {
+        btnCircle[i].innerHTML = '<input type="button" class ="btn-circle" onclick="checkXorO(event)">';
+    }
+    btnsArray = document.getElementsByClassName("btn-circle");
+}
+function checkXorO(event) {
+    playerName = document.getElementById("player-turn");
+    gameStatus = document.getElementById("game-status");
+    count++;
+    flag = false;
+    if (playerTurn === "X") {
+        playerName.innerHTML = player1;
+        event.target.value = "X";
+        if (light === true) {
+            event.target.style.backgroundColor = "#D8E2DC";
```

Comment:
you didn't follow the instructions. you should've changed the whole screen to be "light" or "dark"